bower.json

```diff
@@ -22,19 +22,20 @@
     "output"
```

Comment:
Can you add proper changelog to commit message?

Reply:
Branch which was migrated was behind few chnages , so updated with latest code